tests/test_qubit_device.py

```diff
@@ -363,53 +367,47 @@ def test_correct_conversion_two_states(self, mock_qubit_device, monkeypatch):
                             [
                             (
                               np.array([2, 3, 2, 0, 0]),
-                              np.array([[0, 1, 0, 0],
-                                        [1, 1, 0, 0],
-                                        [0, 1, 0, 0],
-                                        [0, 0, 0, 0],
-                                        [0, 0, 0, 0]],
+                              np.array([[1, 0],
```

Comment:
The comments at 363-365 will also be false.

Reply:
Removed!